src/writer/app_runner.py

```
@@ -1245,6 +1261,52 @@ def reload_code_from_saved(self) -> None:
+                    if self.is_app_process_server_ready.is_set():
+                        self.queue_announcement("codeUpdate", None)
+                        return True
+                    if self.is_app_process_server_failed.is_set():
+                        return False
+                    threading.Event().wait(poll_interval)
+                    elapsed += poll_interval
+                return False
+            else:
+                # Don't wait at all, just start and return
+                return True
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)